pkg/apis/kops/cluster.go

```diff
@@ -384,7 +384,7 @@ type ClusterSubnetSpec struct {
 	Egress string `json:"egress,omitempty"`
 	// Type define which one if the internal types (public, utility, private) the network is
 	Type SubnetType `json:"type,omitempty"`
-	// PublicIP to attatch to NatGateway
+	// PublicIP to attach to NatGateway
 	PublicIP string `json:"publicIP,omitempty"`
```

Comment:
Great finds! Can you add these to the other versions of this file:
https://github.com/kubernetes/kops/blob/master/pkg/apis/kops/v1alpha1/cluster.go
https://github.com/kubernetes/kops/blob/master/pkg/apis/kops/v1alpha2/cluster.go

This helps keeping the docs for the different versions the same!